frontend/src/app/updatesweb/create-modify.component.ts

```diff
@@ -749,6 +740,36 @@ export class CreateModifyComponent implements OnInit, OnDestroy {
         });
     }
 
+    private calculateStatusForInetnumInUpdate() {
+        const statusAttr: IAttributeModel = this.whoisResourcesService.getSingleAttributeOnName(this.attributes, 'status');
+        if (statusAttr.value === 'ALLOCATED PA') {
```

Comment:
This is a custom logic for this specific case.
In this case it doesn't matter if comaintainer or what is the status of the parent object
We always return ALLOCATED-ASSIGNED PA when the resource that we are trying to change is ALLOCATED PA